elastic/beats
# [metricbeat]Docker: add size flag to docker.container

Solves https://github.com/elastic/beats/issues/14979

Docker container metricset is not retrieving size info.


Testing instructions:
- create a docker container
- configura metricbeat enabling docker.container
- make sure `docker.container.size.root_fs` and `docker.container.size.rw` are non 0 values
